docs/use/participate-in-network/index.md

```diff
@@ -1,3 +1,11 @@
+"Participate in Governance"
+---
+id: chain-governance
+order: 8
+title: Participate in Governance
+contributors: <Kate@Centrifuge.io Cassidy Daly:cassidy@centrifuge.io>
```

Comment:
@KateBee-1, make sure to close the email:
`<Kate Beecroft:kate@centrifuge.io>, <Cassidy Daly:cassidy@centrifuge.io>`